CHANGELOG.md

```diff
@@ -29,6 +29,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - `tt coredump pack`: if `-e` option is omitted first search tarantool
   executable in tt environment then in `PATH` instead of using the hardcoded
   path `/usr/bin/tarantool`.
+- `tt replicaset downgrade`: make version a positional argument rather than
+  using the mandatory option `-v` (`--version`).
```

Comment:
So maybe remove the `-v` flag?

Reply:
Do you mean this CHANGELOG entry shouldn't mention it?

Reply:
Uh, I see, you already remove it. Yes, a changelog entry should mention it so - it is not obvious at now.

Reply:
And a TarantoolBot comment too.